src/DynamoUtilities/DynamoFeatureFlagsManager.cs

```diff
@@ -87,12 +99,21 @@ internal T CheckFeatureFlag<T>(string featureFlagKey, T defaultval)
                 return defaultval;
             }
             // if we have not retrieved flags from the cli return empty
-            // and log.
+            // and log once,
            
             if(AllFlagsCache == null)
             {
-                RaiseMessageLogged("the flags cache is null, something went wrong retrieving feature flags," +
-                  " or you need to wait longer for the cache to populate, you can use the static FlagsRetrieved event for this purpose. ");
+                switch (flagsCacheAttempted)
+                {
+                    case true when !nullCacheLogged:
+                        RaiseMessageLogged("The flags cache is null, something went wrong retrieving feature flags. This message will not be logged again, and future calls to CheckFeatureFlags will return default values.");
+                        nullCacheLogged = true;
```

Comment:
no resource strings?

Reply:
no, these logs are intended for internal use.